airbyte-integrations/connectors/source-salesforce/unit_tests/config_builder.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+
+from datetime import datetime
```

Comment:
This will clash with the source-salesforce part of https://github.com/airbytehq/airbyte/pull/36811/files#diff-82cd2ba6c00f795c829bdc86d1f039933a6168b6c1c6c1a4b89226c7b5c6bf67 but that shouldn't be too much of a hassle